samples/Azure.ResourceManager.Sample/Generated/Models/VirtualMachineExtensionUpdate.cs

```diff
@@ -57,7 +57,7 @@ public VirtualMachineExtensionUpdate()
         /// To assign an object to this property use <see cref="BinaryData.FromObjectAsJson{T}(T, System.Text.Json.JsonSerializerOptions?)"/>.
         /// </para>
         /// <para>
-        /// To assign an already formated json string to this property use <see cref="BinaryData.FromString(string)"/>.
+        /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
```

Comment:
There are quite a few files with only this typo fixing change.
Maybe we could open another standalone PR only fix this for every RP, which will reduce the amount of changed files dramatically I believe

Reply:
I will do this and let you know when it is ready

Reply:
Created a PR: https://github.com/Azure/autorest.csharp/pull/3868
Once this is merged to feature/v3 and its corresponding changes merged to azure-sdk-for-net, I believe the diff in your regen should be significantly reduced

Reply:
This change has been merge into main of azure-sdk-for-net. This part of diff should vanish